lib/Model/Importer/Binary/ELFImporter.h

```diff
@@ -0,0 +1,207 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+using namespace llvm;
+
+namespace {
+
+static Logger<> Log("elf-importer");
```

Comment:
Doesn't `revng-check-conventions` complain about this?
There should be no `static` variables in `.h` files: this creates a distinct `Log` object for each translation unit including this file. Keep `Logger` definitions in a single `.cpp` file (*declarations* are fine in `.h` files).

Reply:
I agree.

>Doesn't revng-check-conventions complain about this?
No. I am using it as:
$ orc shell ./scripts/revng-check-conventions --force-format
And it reports nothing. Am I missing something?

Reply:
Need to double check this.